FamilyTree/po/it-local.po

```diff
@@ -19,291 +19,292 @@ msgstr ""
-msgstr ""
+msgstr "(%(description)s)"
 
 #: FamilyTree/FamilyTree.py:1043
 msgid "Tree Options"
-msgstr ""
+msgstr "Opzioni per l'albero"
 
 #: FamilyTree/FamilyTree.py:1045
 msgid "Center Family"
-msgstr ""
+msgstr "Famiglia Principale"
 
 #: FamilyTree/FamilyTree.py:1046
 msgid "The center family for the tree"
-msgstr ""
+msgstr "La famiglia principale dell'albero"
 
 #: FamilyTree/FamilyTree.py:1049
 msgid "Ancestor Generations"
-msgstr ""
+msgstr "Generazioni Ascendenti"
 
 #: FamilyTree/FamilyTree.py:1050
 msgid "The number of ancestor generations to include in the tree"
-msgstr ""
+msgstr "Il numero di generazioni di antenati da includere nell'albero"
 
 #: FamilyTree/FamilyTree.py:1053
 msgid "Descendant Generations"
-msgstr ""
+msgstr "Generazioni Discendenti"
 
 #: FamilyTree/FamilyTree.py:1054
 msgid "The number of descendant generations to include in the tree"
-msgstr ""
+msgstr "Il numero di generazioni di discendenti da includere nell'albero"
 
 #: FamilyTree/FamilyTree.py:1057
 msgid "Kekule number of husband"
-msgstr ""
+msgstr "Numero kekule di mariti"
 
 #: FamilyTree/FamilyTree.py:1058
 msgid ""
 "The Kekule number of the husband (central family). Set 0 to not show Kekule "
 "numbers"
-msgstr ""
+msgstr "Il numero Kekule del marito (famiglia principale). Scegli 0 per non mostrare il Kekule "
+"numbers"
 
 #: FamilyTree/FamilyTree.py:1061
 msgid "Scale to fit on a single page"
-msgstr ""
+msgstr "Ridimensiona su una singola pagina"
 
 #: FamilyTree/FamilyTree.py:1062
 msgid "Whether to scale to fit on a single page."
-msgstr ""
+msgstr "Se ridimensionare per farlo entrare in una sola pagina"
 
 #: FamilyTree/FamilyTree.py:1065
 msgid "Color"
-msgstr ""
+msgstr "Colore"
 
 #: FamilyTree/FamilyTree.py:1067
 msgid "No color"
-msgstr ""
+msgstr "Nessun colore"
 
 #: FamilyTree/FamilyTree.py:1068
 msgid "Generations"
-msgstr ""
+msgstr "Generazioni"
 
 #: FamilyTree/FamilyTree.py:1069
 msgid "First generation"
-msgstr ""
+msgstr "Prima generazione"
 
 #: FamilyTree/FamilyTree.py:1070
 msgid "Second generation"
-msgstr ""
+msgstr "Seconda generazione"
 
 #: FamilyTree/FamilyTree.py:1071
 msgid "Third generation"
-msgstr ""
+msgstr "Terza generazione"
 
 #: FamilyTree/FamilyTree.py:1072
 msgid "Male line"
-msgstr ""
+msgstr "Linea maschile"
 
 #: FamilyTree/FamilyTree.py:1073
 msgid "Male line and illegitimate children"
-msgstr ""
+msgstr "Linea maschile e figli illegittimi"
 
 #: FamilyTree/FamilyTree.py:1074
 msgid "Female line"
-msgstr ""
+msgstr "Linea femminile"
 
 #: FamilyTree/FamilyTree.py:1077
 msgid "Shuffle colors"
-msgstr ""
+msgstr "Mischia i colori"
 
 #: FamilyTree/FamilyTree.py:1078
 msgid "Whether to shuffle colors or order them in rainbow fashion."
-msgstr ""
+msgstr "Se migliare i colori o ordinarli a mo di arcobaleno"
 
 #: FamilyTree/FamilyTree.py:1081
 msgid "Content"
-msgstr ""
+msgstr "Contenuto"
 
 #: FamilyTree/FamilyTree.py:1083
 msgid "Use call name"
-msgstr ""
+msgstr "Usa soprannomi"
```

Comment:
call name is translated as pseudonimo in gramps, please use the same translation (soprannome is used for nickname)